mapusaurus/respondents/views.py

```diff
@@ -87,9 +88,7 @@ def search_results(request):
     
     query = SearchQuerySet().models(Institution).load_all()
     
-    current_sort = request.GET.get('sort')
-    if current_sort == None:
-        current_sort = '-assets'
+    current_sort = '-assets'
```

Comment:
I believe we're still passing a sort parameter that will require 91-92. May want to add back in unless that param is being handled elsewhere.  Thanks!